### PR TITLE
Adds content to HTML Doctype page

### PR DESCRIPTION
**Pull request by Glazy**
Adds a description, code example and provides links to MDN and W3Schools for further reading.

### CODE REVIEW

File: src/pages/html/elements/doctype/index.md [hunk 1]
```diff
@@ -3,13 +3,18 @@ title: Doctype
...
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+```html
+<!DOCTYPE html>
+```
 
-#### More Information:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
+Above you can see an example of an HTML5 doctype. Even though it appears to be a standard HTML tag it actually is not, its a direct instruction to the web browser specifying what version of HTML the page is written in.
```
Comment (jamesjpk123) [inline src/pages/html/elements/doctype/index.md]: You should change "...standard HTML tag it actually is not, its a direct" to "..standard HTML tag it actually is not, **it's** a direct." to fix a grammar error. Great job on the article!

Comment (Glazy) [reply src/pages/html/elements/doctype/index.md]: d8c762d70d71dcf9637693987a67e1083a5a17b0

Comment (Glazy) [reply src/pages/html/elements/doctype/index.md]: Also, thanks for the review @jamesjpk123!

**Comment by ChaituVR** [comment]
Sorry, someone did write this article.
Thank you for your pull request. Please read this [style guide](https://github.com/freeCodeCamp/guides#article-style-guide). I am closing this pull request for now. You can always open a new pull request, Please let me know if you have any questions.

**Pull request closed without merging**